virttest/virsh.py

```diff
@@ -4069,6 +4069,22 @@ def blkdeviotune(name, device=None, options=None,
     return command(cmd, **dargs)
 
 
+def perf(domain="", options="", events="", other_opt="", **dargs):
```

Comment:
as domain is mandate, let us not have domain as default argument.

Reply:
Changed 
def perf(domain="", options="", events="", other_opt="", **dargs):
to 
def perf(domain, options="", events="", other_opt="", **dargs):